docs/news/posts/2025-10-06-migration-overview.md

```diff
@@ -0,0 +1,219 @@
+
+**This will be especially apparent for jobs on pascalnodes and amperenodes.** Prior to the initial compute migration, there will be a 100%:0% split of pascalnodes and 75%:25% split of amperenodes on GPFS4 and GPFS 5, respectively. After the initial compute migration, the splits will be 0%:100% for pascalnodes and 25%:75% for amperenodes on GPFS 4 and GPFS 5, respectively. We understand this will have a large impact on job throughput for individual users and will endeavor to complete the migration swiftly to prevent undue burden on our community. Thank you for understanding.
+
+### Directory Paths
+
+We have reorganized our GPFS 5 filesets to improve file access performance specifically for our network scratch space. As part of this, we have changed the `/data` drive to `/gpfs` and have brought `/scratch` under it.
+
+**Path conversions from GPFS 4 to GPFS 5**:
+
+- `/data/user` -> `/gpfs/user`
+- `/data/user/home` -> `/gpfs/user/home`
+- `/data/project` -> `/gpfs/project`
+- `/scratch` -> `/gpfs/scratch`
+
+In order to smooth the transition to GPFS 5, we have provided symlinks that mimic the GPFS 4 directory structure:
+
+**GPFS 5 symlinks**:
+
+- `/data` -> `/gpfs`
+- `/scratch` -> `/gpfs/scratch`
+- `/home` -> `/gpfs/user/home`
+
+<!-- markdownlint-disable MD046 -->
+!!! critical
+
+    It is **not** necessary to immediately make changes to your paths in your scripts. The provided symlinks ensure backwards compatibility with GPFS 4 path structure in existing scripts in 99% of cases. Requeued and pending jobs submitted using GPFS 4 paths will not be impacted by these path changes.
+<!-- markdownlint-enable MD046 -->
+
+As you use GPFS 5, we suggest switching to the new path structure in your scripts as the symlinks will eventually be deprecated.
+
+### Quotas
+
+Existing quotas on project spaces have not been altered as part of the migration in order to smoothly accommodate existing data. Quotas will be enforced as the sum of storage used across both GPFS and CephFS. Future changes to quotas will be communicated when appropriate.
+
+### Scratch
+
+Network scratch space (`/scratch` or `/gpfs/scratch`) will have a couple of major changes moving forward
+
+1. Total `/scratch` capacity will be reduced from 800 TiB to **500 TiB** for all users. This is a consequence of a necessary reduction in total GPFS capacity.
+1. Scratch policies regarding old data will be enforced. Files older than 30 days will be automatically offloaded for deletion.
+1. Scratch does not obey the same tiered storage principles as explained in the [data tiering](#gpfs-5-data-tiering) section. Data offloaded from scratch due to age will be subject to deletion as opposed to indefinite storage on Ceph.
+
+Please be cognizant of these changes moving forward and remember that `/gpfs/scratch` is a shared storage space.
```

Comment:
`/scratch`  is a shared storage space that is limited by total capacity.

We will continue to adjust scratch policies based on application usage and performance requirements.